pkg/utils/has/controller.go

```diff
@@ -270,14 +270,9 @@ func (h *SuiteController) ComponentDeleted(component *appservice.Component) wait
 }
 
 // CreateComponentWithPaCEnabled creates a component with "pipelinesascode: '1'" annotation that is used for triggering PaC builds
-func (h *SuiteController) CreateComponentWithPaCEnabled(applicationName, componentName, namespace, gitSourceURL, baseBranch string, deleteRepo bool) (*appservice.Component, error) {
+func (h *SuiteController) CreateComponentWithPaCEnabled(applicationName, componentName, namespace, gitSourceURL, baseBranch string) (*appservice.Component, error) {
```

Comment:
this function is being used [here](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go#L262) as well.

Reply:
Yes. `ci/prow/redhat-appstudio-e2e` is failing because of that: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/602/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1671762470873600000#1:build-log.txt%3A1053

Reply:
When I was creating this PR, that file wasn't existing. Thank you.